raiden/network/transport/matrix.py

```diff
@@ -127,14 +128,25 @@ class MatrixTransport:
     def __init__(self, config: dict):
         self._bound_logger = None
         self._raiden_service: RaidenService = None
+
+        def http_retry_delay() -> Union[float, Iterable[float]]:
```

Comment:
Isn't this a private method (i.e. should be `_http_retry_delay()`)?

Reply:
No, it isn't a method, it's just a contextual function, so as it isn't exposed at all, I don't think there's need to rename it differently. Anyway, even if it doens't matter, I think it's nice to sign that this function isn't to anyone to look into, so will rename =)